allium/lib/aroi_validation.py

```
@@ -94,6 +94,21 @@ def get_proof_type_version(proof_type):
 
 
+def get_v3_search_index_thresholds() -> Dict[str, int]:
+    """Return v3 tier thresholds in search-index JSON units.
+
+    The search index exposes percentages as integers because the Pages
+    Function is JavaScript and should not duplicate Allium's float constants.
+    ``explorer`` is the one count-based threshold: at least one v3 relay.
+    """
+    return {
+        'explorer': 1,
+        'migrating': int(round(V3_TIER_MIGRATING * 100)),
```

Ruff (0.15.20)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)